core-storage/src/main/java/com/thoughtworks/ark/core/storage/FileManager.kt

```diff
@@ -0,0 +1,116 @@
+package com.thoughtworks.ark.core.storage
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.os.Environment
+import android.os.StatFs
+import com.thoughtworks.ark.core.logging.Logger
+import java.io.File
+import java.io.IOException
+
+class FileManager : StorageInterface {
+
+    override var path: String? = externalBaseDir
+
+    // It is only works less than target sdk 29
```

Comment:
Use code rather than a comment to explicit implement behaviour
```
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
   // code for Lollipop and later
}
```

Reply:
get it